mhs/inbound/main.py

```diff
@@ -85,10 +85,20 @@ def start_inbound_server(local_certs_file: str, ca_certs_file: str, key_file: st
     healthcheck_application = tornado.web.Application([
         ("/healthcheck", healthcheck_handler.HealthcheckHandler)
     ])
-    healthcheck_application.listen(80)
+    server_port = int(config.get_config('INBOUND_SERVER_PORT', default='80'))
+    healthcheck_application.listen(server_port)
 
-    logger.info('011', 'Starting inbound server')
-    tornado.ioloop.IOLoop.current().start()
+    logger.info('011', 'Starting inbound server at port {server_port}', {'server_port': server_port})
+    tornado_io_loop = tornado.ioloop.IOLoop.current()
+    try:
+        tornado_io_loop.start()
+    except KeyboardInterrupt:
```

Comment:
To silence the red log entry when you kill app in pycharm

Reply:
I know it's a duplicate between all 3 servers but it might be hard to change that beacuse of the fcnkg logging lib

Reply:
just a question, because it's out of the scope of this task - is it a graceful shutdown?
And should we even care about ongoing requests?

Reply:
no it's not
I tried doing the graceful shitdown using this https://github.com/tornadoweb/tornado/issues/1791
but looks like we use a different version of tornado so it's missing some pieces